File: frontend/src/types/ScheduleDetail.ts
import Film from './Film';
import Room from './Room';
import Ticket from './Ticket';

export default interface ScheduleDetail {
    scheduleId: string,
    room: Room,             // Phòng sử dụng của lần chiếu này
    film: Film,             // Bộ phim được chiếu
    startTime: Date,        // Thời gian bắt đầu chiếu
    endTime: Date           // Thời gian kết thúc của lần chiếu này
    tickets: Array<Ticket>
}

File: backend/Models/ScheduleDetail.cs
using Cinema.Entities;

namespace Cinema.Models;

public class ScheduleDetail
{
    public string ScheduleId { get; set; }

    public Room Room { get; set; }

    public Film Film { get; set; }

    public DateTime StartTime { get; set; }

    public DateTime EndTime { get; set; }

    public IEnumerable<Ticket> Tickets { get; set; }
}
